services/core-web/src/components/modalContent/NoPermitRequiredSelectionModal.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { Alert, Select, Popconfirm, Button } from "antd";
+
+const propTypes = {
+  closeModal: PropTypes.func.isRequired,
+  nextStep: PropTypes.func.isRequired,
+  signature: PropTypes.string.isRequired,
+};
+
+const defaultProps = {};
+
+export const NoPermitRequiredSelectionModal = (props) => {
+  const [letterCode, setLetterCode] = useState(null);
+  const [disabled, setDisabled] = useState(true);
+
+  const options = [
+    { value: "NPR", label: "No Permit Required" },
+    { value: "NPI", label: "No Permit Required IP" },
+  ];
+
+  useEffect(() => {
+    if (letterCode) {
+      setDisabled(false);
+    }
+  }, [letterCode]);
+
+  const handleNextStep = () => props.nextStep(letterCode);
+
+  return (
+    <div>
+      {!props.signature && (
+        <>
+          <Alert
+            message="Signature needed to generate."
+            description="The signature for the Issuing Inspector has not been provided."
+            type="error"
+            showIcon
+          />
+          <br />
+        </>
+      )}
+      <Select
+        virtual={false}
+        labelInValue
+        placeholder="Select a No Permit Required Letter..."
```

Comment:
Can we remove the ellipsis? Don't usually see this in our placeholder text. Maybe "Select Letter Type"?

Reply:
Done.